locales/ku_KU/duckduckgo.po

```diff
@@ -0,0 +1,5938 @@
+msgctxt "welcome message eu search preference android"
+msgid "Block unsafe trackers"
+msgstr "Şopgerên neparastî asteng bike"
+
+#. shown after the DuckDuckGo extension is installed
+msgctxt "welcome message"
+msgid "Blocks trackers on websites you visit"
+msgstr "li ser malperên ku tu serdan dikî şopgeran asteng dike "
+
+msgid "Blog"
+msgstr "Blog"
+
+#. The color blue
+msgid "Blue"
+msgstr "Şîn"
+
+msgctxt "color"
+msgid "Blue"
+msgstr "Şîn"
+
+#. Click "Show Bookmarklet and Settings Data" - http://i.imgur.com/WyGptiV.png
+#. In the box that opens - http://i.imgur.com/tZAkHF6.png
+msgctxt "settings"
+msgid "Bookmarklet URL:"
+msgstr "Şûnpela URL:"
+
+#. The name of the Bosnian language
+msgctxt "language_name"
+msgid "Bosnian"
+msgstr "Boşnaqî"
+
+msgid "Brazil"
+msgstr "Brazîlya"
+
+msgctxt "color"
+msgid "Brown"
+msgstr "Qehweyî"
+
+msgid ""
+"Browse as usual while we add privacy protection. We bundled our search "
+"engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
+msgstr "Geranê xwe wekî her dem bike û em ê jî parastina taybetiyê tevlî bikin. Me motora lêgerînê xwe, astengkera şopgeran, û şîfrekirin di yek %s%sPêvek de pêk anî."
+
+msgid ""
+"Browse as usual, and we'll take care of the rest. We bundled our search "
+"engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
+msgstr "Geranê xwe wekî her dem bike, û em ê yên mayî li ser me ye. Me motora lêgerînê xwe, astengkera şopgeran, û şîfrekirin di yek %s%sPêvek de pêk anî."
+
+msgid ""
+"Browse as usual, and we’ll take care of the rest. Get bundled private "
+"search, tracker blocking, and site encryption, all in one download, for "
+"%smajor browsers%s."
+msgstr "Geranê xwe wekî her dem bike, û em ê yên mayî li ser me ye. Paketa lêgerîna parastî, astengkera şopgeran, û şîfrekirina malperan di yek daxistin de bi dest bixe, ji bo "%sgerokên sereke%s."
+
+msgctxt "precise_user_location"
+msgid "Browser Location Unavailable"
+msgstr "Lêgerînê cihê ne berdest e"
+
+msgctxt "settingsvalue"
+msgid "Browser Preferred Language"
+msgstr "Zimanê gerokê hilbijartî"
+
+msgctxt "settingsvalue"
+msgid "Browser preferred language"
+msgstr "Zimanê gerokê hilbijartî"
+
+msgid "Bulgaria"
+msgstr "Bulgarîstan"
+
+#. The name of the Bulgarian language
+msgctxt "language_name"
+msgid "Bulgarian"
+msgstr "Bulgarî"
+
+#. Click "Show Bookmarklet and Settings Data" - http://i.imgur.com/NKOWrRl.png
+#. Then - http://i.imgur.com/mo6wsE4.png
+msgctxt "settings"
+msgid "sazkarî"
+"By default, settings are stored in non-personal browser cookies (in your "
+"browser). You can use anonymous Cloud Save to store your settings in a more "
+"permanent way (on a remote server in the cloud). No personally identifiable "
+"information will be stored in the cloud, and your passphrase will never "
+"leave your browser."
+msgstr "Bi berdestî, sazkarî wê werin tomarkirin li ser kilorên gerokê yên ne kesane (di "gerokê" te de). 
```

Comment:
Please do not use double quotes within the strings. It will break the format and make the file unusable. I suggest you use a specified .po editor like https://poedit.net/ (there is a free version). It will prevent such problems.